FormalConjectures/ErdosProblems/678.lean

```diff
@@ -0,0 +1,74 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+
+/-!
+# Erdős Problem 678
+*Reference:* [erdosproblems.com/678](https://www.erdosproblems.com/678)
+-/
+
+open Asymptotics Filter Finset
+
+/--
+Write $M(n, k)$ be the least common multiple of ${n+1, \dotsc, n+k}$, denoted here as
+`lcmInterval n k`.
+-/
+def lcmInterval (n k : ℕ) : ℕ := (Finset.Ioc n (n + k)).lcm id
+
+/--
+The referee of [Er79] found the example $M(96, 7) > M(104, 8)$, showing that there are cases where
+$M(n, k) > M(m, k + 1)$ with $m \geq n + k$.
+[Er79] Erdős, Paul, Some unconventional problems in number theory. Math. Mag. (1979), 67-70.
+-/
+@[category API]
```

Comment:
Here I'd be tempted to put `test` as the category, but judging by the docstring this is a useful (but easy) problem. What do you think of having `high_school` instead?